compose/settings.gradle.kts

```
@@ -0,0 +1,42 @@
+pluginManagement {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven("https://cboost.jfrog.io/artifactory/chartboost-mediation/") {
+            name = "Chartboost Mediation's Production Repo"
+        }
+        maven("https://cboost.jfrog.io/artifactory/chartboost-ads/") {
+            name = "Chartboost's maven repo"
+        }
+
+        maven("https://sdk.tapjoy.com/") {
+            name = "Tapjoy's maven repo"
+        }
+        maven("https://android-sdk.is.com/") {
+            name = "IronSource's maven repo"
+        }
+        maven("https://dl-maven-android.mintegral.com/repository/mbridge_android_sdk_oversea") {
+            name = "Mintegral's maven repo"
+        }
+        maven("https://artifactory.verizonmedia.com/artifactory/maven/") {
+            name = "Yahoo's maven repo"
+        }
+        maven("https://artifact.bytedance.com/repository/pangle") {
+            name = "Pangle's maven repo"
+        }
+        maven("https://verve.jfrog.io/artifactory/verve-gradle-release") {
+            name = "Verve's maven repo"
+        }
+    }
+}
+
+rootProject.name = "ChartboostMediationDemo"
+include(":app")
```

Can we have the module to be called  `ChartboostMediationDemo` or `ChartboostMediationDemoApp`?
```suggestion
include(":ChartboostMediationDemo")
```